.github/workflows/lint.yml

```
@@ -0,0 +1,31 @@
+        with:
+          node-version: 22
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Svelte check
+        run: pnpm run check
+
+      - name: Lint
+        run: pnpm run check
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)